node/test/graphql/query/me/training-data.spec.ts

```diff
@@ -115,6 +115,27 @@ describe('training data', () => {
     expect(response.body.data.me.trainingData.isTrainable).to.eql(false);
   });
 
+  it(`does not return invalidated training data`, async () => {
```

Comment:
see comment above, let's just tweak this test to actually use the invalidate function (rather than just test data that was marked invalidated in the mongodata)